paseos/tests/event_based_test.py

```diff
@@ -0,0 +1,53 @@
+"""Simple test for the operations monitor"""
+
+import numpy as np
+import pykep as pk
+
+import paseos
+from paseos import ActorBuilder, SpacecraftActor, load_default_cfg
+
+
+async def test_event_based_mode():
```

Comment:
shall we add a test to see if events and activities are compatible?

Reply:
they aren't though 🥲